toolsets/oracle-operator/primitive.go

```diff
@@ -28,46 +27,29 @@ func queryPrimitive(
 ) {
 	logger := ctx.Logger()
 	logger.Debug("query primitive")
-	retBool, retString, err := callContract(
-		ctx,
-		primitive.PrimitiveContractAddr,
-		types.PrimitiveContractFnName,
-		[]string{payload.Contract, payload.Function},
+	logger.Debug(payload.Contract, payload.Function)
```

Comment:
Need a "function" key for the second argument. `logger.Debug(payload.Contract, "function", payload.Function)`